Target-MVD-Mathias/Extensions/UILabelExtension.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  UILabelExtension.swift
+//  Target-MVD-Mathias
+//
+//  Created by Mathias Cabano on 8/16/17.
+//  Copyright © 2017 TopTier labs. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UILabel {
+  
+  func setSpacing(ofLine lineSpacing: CGFloat? = nil, ofCharacter character: CGFloat) {
+    if let mutableAttributes = attributedText?.mutableCopy() as? NSMutableAttributedString, let text = text {
+  
+      if let lineSpacing = lineSpacing {
```

Comment:
here you don't want to step over an existing paragraph style. @aguscha333 made an important update to this helper that you could use. 😉